src/System Application/App/Email/src/Message/EmailMessageImpl.Codeunit.al

```diff
@@ -695,6 +738,24 @@ codeunit 8905 "Email Message Impl."
     begin
     end;
 
+    [EventSubscriber(ObjectType::Table, Database::"Email Inbox", OnBeforeInsertEvent, '', false, false)]
+    local procedure OnBeforeInsertEventEmailInbox(var Rec: Record "Email Inbox"; RunTrigger: Boolean)
+    begin
+        Rec."User Security Id" := UserSecurityId();
+    end;
```

Comment:
Did you intentionally omit the Rec.IsTemporary check?

Reply:
It wasn't suppose to be left out, thanks.